RELEASING/release-notes-2-0/changelog.md

```diff
@@ -314,7 +314,7 @@ under the License.
 - [#19166](https://github.com/apache/superset/pull/19166) fix: time filter should be [start, end) (@zhaoyongjie)
 
 **Others**
-- [#20116](https://github.com/apache/superset/pull/20116) style(typo): occured -> occurred (@sfirke)
+- [#20116](https://github.com/apache/superset/pull/20116) style(typo): occurred -> occurred (@sfirke)
```

Comment:
Sigh, I'll drop this and add a pattern to try to guard against spotting it again

Reply:
Yeah, this is an odd case where we'd _want_ the typo.